worker/src/task/2-submission.ts

```
@@ -1,102 +1,187 @@
+  [key: string]: any;
+}
+
+interface SubmissionParams {
+  orcaClient: any;
+  roundNumber: number;
+  stakingKey: string;
+  publicKey: string;
+  secretKey: Uint8Array<ArrayBufferLike>;
+}
+
+export async function submission(roundNumber: number): Promise<string | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)